src/main/webapp/query-control.js

```diff
@@ -76,47 +76,86 @@ function putHistory(personType, action, location, year, userId) {
   });
 }
 
+function createSlide(title, url, location) {
+  const historyItem = document.createElement('li');
+  historyItem.className = 'splide__slide';
+  const link = document.createElement('a');
+  link.href = url;
+  const titleNode = document.createTextNode(title);
+  const imgWrapper = document.createElement('div');
+  imgWrapper.className = 'img-wrapper';
+  const image = document.createElement('img');
+  image.alt = 'Small U.S. map';
```

Comment:
now that the map is different depending on location, you could append ' of ' + stateInfo[location].name for county queries